hostscripts/soc-ci/soc-ci

```diff
@@ -71,6 +71,12 @@ def _config_get():
         conf.set(section_name, "artifacts_dir",
                  os.path.expanduser('~/.soc-ci/artifacts/'))
 
+    # SSL verification
+    ssl_verify = True
+    if conf.has_option(section_name, "ssl_verify"):
+        ssl_verify = conf.getboolean(section_name, "ssl_verify")
+    conf.set(section_name, "ssl_verify", ssl_verify)
```

Comment:
Hm. This is setting ```ssl_verify``` every time. Is that needed?
Wouldn't something like
```
if not conf.has_option(section_name, "ssl_verify"):
    conf.set(section_name, "ssl_verify", True)
```
make more sense? In that case, it would set the default to ```True``` if it is not there yet. Otherwise it would just be read from the config file

Reply:
It is, or something like it. If we read it from the config file the value is a string, which we need to cast to a boolean, or stuff breaks. We can do the above and make use of getboolean(), I can refactor it with a cast. I'm happy to add in a comment to ease confusion if you think that is required.

Reply:
Hm. Or ```_config_get()``` should just return the ```conf``` object and we use the correct getter methods in the code.

Reply:
Also, section_name is currently confined to _config_get(), that change means the whole script needs to learn about it.

Reply:
@s-t-e-v-e-n-k done in https://github.com/SUSE-Cloud/automation/pull/2398